draft-ietf-quic-version-negotiation.md

```diff
@@ -96,14 +117,11 @@ This document specifies two means of performing version negotiation: one
 and one "compatible" that allows saving the round trip but only applies when the
 versions are compatible.
 
-The client initiates a QUIC connection by choosing an initial version and
+The client initiates a QUIC connection by choosing an original version and
```

Comment:
This should still say "initial" because incompatible VN is two different connections.

Reply:
But this sentence only refers to the first connection. Maybe we need a better name that refers to the set of related connections.

Reply:
I don't get that out of this text. 

The client initiates a QUIC connection. It gets a VN. It then initiates a new QUIC connection.

Reply:
That's only true of the first connection, see Section 4:
> A client that makes a connection attempt based on information received from a Version Negotiation packet MUST ignore any Version Negotiation packets it receives in response to that connection attempt.

Reply:
That statement is true, but irrelevant to this sentence.

Reply:
Sure, I don't feel too strongly
```suggestion
The client initiates a QUIC connection by choosing an initial version and
```